extensions/amp-facebook-comments/0.1/amp-facebook-comments.js

```diff
@@ -0,0 +1,78 @@
+/**
```

Comment:
Please also add `amp-facebook-comments.md` as some other extensions do and reference https://developers.facebook.com/docs/plugins/comments